docs/getting-started-guides/rkt/index.md

```diff
@@ -135,52 +130,26 @@ Then you can launch the cluster by:
 $ cluster/kube-up.sh
 ```
 
-Note that we are still working on making all containerized the master components run smoothly in rkt. Before that we are not able to run the master node with rkt yet.
-
 ### CoreOS cluster on AWS
 
-To use rkt as the container runtime for your CoreOS cluster on AWS, you need to specify the provider and OS distribution:
-
-```shell
-$ export KUBERNETES_PROVIDER=aws
-$ export KUBE_OS_DISTRIBUTION=coreos
-$ export KUBE_CONTAINER_RUNTIME=rkt
-```
-
-You can optionally choose the version of rkt used by setting `KUBE_RKT_VERSION`:
-
-```shell
-$ export KUBE_RKT_VERSION=0.8.0
-```
-
-You can optionally choose the CoreOS channel  by setting `COREOS_CHANNEL`:
-
-```shell
-$ export COREOS_CHANNEL=stable
-```
-
-Then you can launch the cluster by:
-
-```shell
-$ kube-up.sh
-```
-
-Note: CoreOS is not supported as the master using the automated launch
-scripts. The master node is always Ubuntu.
+`kube-up` for AWS is currently supported.
+Instead, we recommend you to refer the [Kubernetes on AWS guide](https://coreos.com/kubernetes/docs/latest/kubernetes-on-aws.html) to launch a CoreOS/rkt cluster on AWS.
 
 ### Getting started with your cluster
 
 See [a simple nginx example](/docs/user-guide/simple-nginx) to try out your new cluster.
 
 For more complete applications, please look in the [examples directory](https://github.com/kubernetes/kubernetes/tree/{{page.githubbranch}}/examples/).
 
-### Different UX with rkt container runtime
+### Known Issues and Differences
 
-rkt and Docker have very different designs, as well as ACI and Docker image format. Users might experience some different experience when switching from one to the other. More information can be found [here](/docs/getting-started-guides/rkt/notes/).
+rkt and Docker have very different designs, as well as ACI and Docker image format.
```

Comment:
@tmrt Could you give me an example? Since OCI is not supported in rkt yet, so I don't know what to say about it, maybe next time?